lib/webpack_manifest/rails/rspec/compilable.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'logger'
+
+module WebpackManifest::Rails
+  module RSpec
```

Comment:
The method in this module is expected to be run by RSpec. But the module name `RSpec` is suitable? 🤔I thought that the method may be defined as a rake task or something without `RSpec` context.